src/Microsoft.ML.Dnn/DnnCatalog.cs

```diff
@@ -85,33 +83,50 @@ public static class DnnCatalog
-        /// <param name="checkpointName">The name of the prefix for checkpoint files.</param>
         /// <param name="arch">The architecture of the image recognition DNN model.</param>
-        /// <param name="dnnFramework">The backend DNN framework to use, currently only Tensorflow is supported.</param>
-        /// <param name="epoch">Number of training epochs.</param>
+        /// <param name="epoch">Number of training iterations. Each iteration/epoch refers to one pass over the dataset.</param>
         /// <param name="batchSize">The batch size for training.</param>
         /// <param name="learningRate">The learning rate for training.</param>
+        /// <param name="metricsCallback">Callback for reporting model statistics during training phase.</param>
+        /// <param name="statisticFrequency">Indicates the frequency of epochs at which to report model statistics during training phase.</param>
+        /// <param name="framework">Indicates the choice of DNN training framework. Currently only tensorflow is supported.</param>
+        /// <param name="modelSavePath">Optional name of the path where a copy new graph should be saved. The graph will be saved as part of model.</param>
+        /// <param name="checkpointName">The name of the prefix for checkpoint files.</param>
+        /// <param name="validationSet">Validation set.</param>
+        /// <param name="testOnTrainSet">Indicates to evaluate the model on train set after every epoch.</param>
+        /// <param name="reuseTrainSetBottleneckCachedValues">Indicates to not re-compute cached trainset bottleneck values if already available in the bin folder.</param>
+        /// <param name="reuseValidationSetBottleneckCachedValues">Indicates to not re-compute validataionset cached bottleneck validationset values if already available in the bin folder.</param>
+        /// <param name="trainSetBottleneckCachedValuesFilePath">Indicates the file path to store trainset bottleneck values for caching.</param>
+        /// <param name="validationSetBottleneckCachedValuesFilePath">Indicates the file path to store validationset bottleneck values for caching.</param>
         /// <remarks>
         /// The support for image classification is under preview.
         /// </remarks>
-        public static DnnEstimator ImageClassification(
+        public static ImageClassificationEstimator ImageClassification(
             this ModelOperationsCatalog catalog,
             string featuresColumnName,
             string labelColumnName,
-            string outputGraphPath = null,
             string scoreColumnName = "Score",
             string predictedLabelColumnName = "PredictedLabel",
-            string checkpointName = "_retrain_checkpoint",
-            Architecture arch = Architecture.ResnetV2101,
-            DnnFramework dnnFramework = DnnFramework.Tensorflow,
+            Architecture arch = Architecture.InceptionV3,
             int epoch = 10,
```

Comment:
Why epoch=10 and batchSize=20 as defaults?
I find that for small/medium datasets (hundres of images) which is what initially folks are going to test, these defaults are what we were using:

epoch=100 
batchSize=10

But I'm open to different/better defaults. IT is simply that I think these defaults were defined in previous versions and we might want to change it?

Of course, I know that we cannot have good defaults for any scenario. We'd need to have some "AutoML" that would change dynamically depending on the input datasets type/size and even other hyper-parameters depending on the accuracy we're getting...